### PR TITLE
create the routes files

### PR DESCRIPTION
**Pull request by A00020061**
Hey bro, I created all components' and pages' files and imported them in App.js. I don't have an idea about the theme provider. Please give me an idea of what I need to do in it.

### CODE REVIEW

**Comment by A00020061** [comment]
setup the app structure by creating components and pages files

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (yoloOnTheBattlefield) [inline .vscode/launch.json:4]: this is not allowed

File: src/App.js [hunk 2]
```diff
@@ -12,30 +16,19 @@ export default function App() {
...
+    
           </Route>
-          <Route path="/users">
-            <Users />
+          <Route path="" exact component={PortfolioPage}>
+            
           </Route>
-          <Route path="/">
-            <Home />
+          <Route path="" exact component = {CoinPage}>
```
Comment (yoloOnTheBattlefield) [inline src/App.js:30]: remove the spaces

File: src/App.js [hunk 3]
```diff
@@ -12,30 +16,19 @@ export default function App() {
...
+            
           </Route>
```
Comment (yoloOnTheBattlefield) [inline src/App.js:32]: route is self enclosing

File: src/Components/Navbar/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {Container} from './index.styles'
+class Navbar extends React.Component{
```
Comment (yoloOnTheBattlefield) [inline src/Components/Navbar/index.js:3]: new line

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 2 above)
Comment (A00020061) [reply src/App.js:30]: I removed the extra spaces

File: src/App.js (hunk 3 above)
Comment (A00020061) [reply src/App.js:32]: Fixed that

File: src/Components/Navbar/index.js (hunk 4 above)
Comment (A00020061) [reply src/Components/Navbar/index.js:3]: defined class after giving some lines of spaces

**Pull request merged**